packages/fxa-content-server/app/scripts/views/pair/index.js

```diff
@@ -130,6 +131,13 @@ class PairIndexView extends FormView {
     this.metrics.logEvent('screen.pair.notnow.engage');
```

Comment:
Do we need to move this to the new handler function?

Reply:
Nope, this is for a different element in the mustache file (see other comment)